jekyll/_cci2/index.md

```diff
@@ -11,12 +11,12 @@ Welcome to CircleCI 2.0 Beta! The Beta release of CircleCI 2.0 includes many imp
 
 Code that builds on Linux will generally build on CircleCI 2.0. For some language versions, CircleCI provides demo applications and Docker images for primary container use, as follows: 
 
-- **Clojure 1.8 and later**, see <https://github.com/circleci/cci-demo-clojure> and <https://hub.docker.com/r/circleci/clojure>
-- **Go 1.7 and later**, see <https://github.com/circleci/cci-demo-go> and <https://hub.docker.com/r/circleci/golang>
-- **Node.js 4 and later**, see <https://github.com/circleci/cci-demo-react> and <https://hub.docker.com/r/circleci/node>
-- **PHP 5 and later**, see <https://github.com/circleci/cci-demo-lumen> and <https://hub.docker.com/r/circleci/php>
-- **Python 2 and later**, see <https://github.com/circleci/cci-demo-flask> and <https://hub.docker.com/r/circleci/python>
-- **Ruby 2 and later**, see <https://github.com/circleci/cci-demo-rails> and <https://hub.docker.com/r/circleci/ruby> 
+- **Clojure 1.8 and later**, see <https://github.com/CircleCI-Public/circleci-demo-clojure-luminus> and <https://hub.docker.com/r/circleci/clojure>
```

Comment:
Minor nitpick: there's no need to specify a version of Clojure we support; any version will work.